src/components/CareCard/CareCard.scss

```diff
@@ -0,0 +1,15 @@
+.CareCard__item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 180px;
+  height: 180px;
+  border-radius: 5.45625px;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  margin: 10px;
+  padding: 10px;
```

Comment:
I think you have to add a background color to these items because when the dark mode is active this is lost.
![55E98311-8503-4D8F-B1E7-A056E1365DB0](https://user-images.githubusercontent.com/3743350/90090483-c04d3f00-dce9-11ea-8415-45cf59519e91.png)
```suggestion
  padding: 10px;
  background-color: white;
```